shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integratedtest/http/SignTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.shenyu.integratedtest.http;
+
+import com.google.common.collect.Maps;
+import com.google.gson.reflect.TypeToken;
+import org.apache.shenyu.common.utils.SignUtils;
+import org.apache.shenyu.integratedtest.http.dto.AdminResponse;
+import org.apache.shenyu.integratedtest.http.dto.UserDTO;
+import org.apache.shenyu.integratedtest.http.helper.HttpHelper;
+import org.junit.Test;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.Map;
+import java.util.concurrent.Future;
+import static org.junit.Assert.assertEquals;
+
+public class SignTest extends AbstractTest{
```

Comment:
Can you reformat the code style?
```suggestion
public class SignTest extends AbstractTest {
```
The same as other places. Thanks.

Reply:
I have formatted the code style, please review it again.